arch/sim/src/sim/up_head.c

```diff
@@ -105,7 +105,7 @@ static void allsyms_relocate(void)
  *
  ****************************************************************************/
 
-__attribute__ ((visibility("default")))
+import_symbol
```

Comment:
let's add this option to HOSTFLAGS:
```
-fvisibility=default
```
and remove all __attribute__ ((visibility("default"))) directly.

Reply:
Done